setup.py

```diff
@@ -59,7 +59,7 @@ def write_version_py(filename=None):
     long_description=README_TEXT,
     packages=find_packages(),
     package_data={'': ['simplemesh.cuds']},
-    install_requires=['simphony == 0.1.3',
```

Comment:
`simphony == 0.2` could be changed to `simphony >= 0.2.0` as your engine has `get_dataset_names` provided in '0.2.1'   (https://www.python.org/dev/peps/pep-0440/#version-specifiers).